pyocd/target/builtin/target_LPC55S36.py

```diff
@@ -0,0 +1,119 @@
+    0x3302f2c1, 0x4718681b, 0x1200f64f, 0x3202f2c1, 0x47106812, 0x1c18f64f, 0x3c02f2c1, 0xc000f8dc,
+    0xea404760, 0xb5100301, 0xd10f079b, 0xd30d2a04, 0xc908c810, 0x429c1f12, 0xba20d0f8, 0x4288ba19,
+    0x2001d901, 0xf04fbd10, 0xbd1030ff, 0x07d3b11a, 0x1c52d003, 0x2000e007, 0xf810bd10, 0xf8113b01,
+    0x1b1b4b01, 0xf810d107, 0xf8113b01, 0x1b1b4b01, 0x1e92d101, 0x4618d1f1, 0x0000bd10, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000
+    ],
+
+    # Relative function addresses
+    'pc_init': 0x20000005,
+    'pc_unInit': 0x20000061,
+    'pc_program_page': 0x200000b5,
+    'pc_erase_sector': 0x2000008d,
+    'pc_eraseAll': 0x20000065,
+
+    'static_base' : 0x20000000 + 0x00000004 + 0x000002dc,
+    'begin_stack' : 0x20000500,
+    'begin_data' : 0x20000000 + 0x1000,
+    'page_size' : 0x200,
+    'analyzer_supported' : False,
+    'analyzer_address' : 0x00000000,
+    'page_buffers' : [0x20001000, 0x20001200],   # Enable double buffering
+    'min_program_length' : 0x200,
+
+    # Relative region addresses and sizes
+    'ro_start': 0x0,
+    'ro_size': 0x2dc,
+    'rw_start': 0x2dc,
+    'rw_size': 0x4,
+    'zi_start': 0x2e0,
+    'zi_size': 0x44,
+
+    # Flash information
+    'flash_start': 0x0,
+    'flash_size': 0x40000,
+    'sector_sizes': (
+        (0x0, 0x8000),
+    )
+}
+
+
+class LPC55S36(LPC5500Family):
+
+    MEMORY_MAP = MemoryMap(
+        FlashRegion(name='nsflash',     start=0x00000000, length=0x040000, access='rx',
+            page_size=0x200,
+            sector_size=0x8000,
+            is_boot_memory=True,
+            are_erased_sectors_readable=False,
+            algo=FLASH_ALGO),
+        RomRegion(  name='nsrom',       start=0x03000000, length=0x020000, access='rx'),
+        RamRegion(  name='nscoderam',   start=0x04000000, length=0x4000, access='rwx',
+            default=False),
+        FlashRegion(name='sflash',      start=0x10000000, length=0x040000, access='rx',
+            page_size=0x200,
+            sector_size=0x8000,
+            is_boot_memory=True,
+            are_erased_sectors_readable=False,
+            algo=FLASH_ALGO,
+            alias='nsflash'),
+        RomRegion(  name='srom',        start=0x13000000, length=0x020000, access='srx',
+            alias='nsrom'),
+        RamRegion(  name='scoderam',    start=0x14000000, length=0x4000, access='srwx',
+            alias='nscoderam',
+            default=False),
+        RamRegion(  name='nsram',       start=0x20000000, length=0x01c000, access='rwx'),
+        RamRegion(  name='sram',        start=0x30000000, length=0x01c000, access='srwx',
+            alias='nsram'),
+        )
+
+    def __init__(self, session):
+        super(LPC55S36, self).__init__(session, self.MEMORY_MAP)
```

Comment:
For future reference, you no longer need to pass parameters to `super()` because pyocd no longer needs to support Python 2.